cmd/libs/go2idl/client-gen/test_apis/testgroup.k8s.io/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +groupNameInURL=testgroup.k8s.io
```

Comment:
The groupName is a fundamental property - why not make it explicit?
If this has to generate error messages in the future, it would use
groupName as well.